examples/misc/01_ader_fd.ipynb

```diff
@@ -0,0 +1,247 @@
+{
```

Comment:
<p>Isn't that second order just the OT4 acoustic kernel but for the first order equations?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Same idea I think</p><p> </p>